client/my-sites/themes/index.node.js

```diff
@@ -11,7 +11,7 @@ import { makeLayout } from 'controller';
 export default function( router ) {
 	if ( config.isEnabled( 'manage/themes' ) ) {
 		router( '/design', makeLayout );
```

Comment:
I think you can now even omit this line since it's covered by the next (`page` is clever enough to take care of the trailing slash, or lack thereof IIRC).